pkg/core/region.go

```diff
@@ -754,18 +753,18 @@ func GenerateRegionGuideFunc(enableLog bool) RegionGuideFunc {
 		if logRunner != nil {
 			debug = func(msg string, fields ...zap.Field) {
 				logRunner.RunTask(
-					ctx.Context,
+					region.GetID(),
```

Comment:
It's better not to pass exclusive attributes to the runner, other types of tasks can also use it.